CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+16.0.1 - 2021-11-24
```

Comment:
Per chat, let's move this to next week and update the date here.

Reply:
Code freeze is now over, please can you review/approve?

Reply:
Could you update the date on line 1?

Thanks

Reply:
Done!